### DISCUSSION TITLE
Tiny Tina's Wonderland - Not able to save settings

### DISCUSSION THREAD

**Discussion by TheodoreLizard** [Heroic-Games-Launcher/HeroicGamesLauncher#3703] [Q&A]
First - my apologies if this has been asked. I searched here, but perhaps used the wrong query as I didn't find a topic.

Running Tiny Tina's Wonderland on Mac using Epics version and CrossOver. Game launches and runs, but I get a message every few seconds saying something along the lines of "Cannot save settings, retry?"

I can play the game, but quitting and resuming just starts the game over since it wasn't saved.

Other games seem to work and save when using CrossOver bottle (right terminology?). So I'm guessing some setting or config needs to be adjusted.

Ideas?

Thanks!

**Comment by trolles** [comment]
Did you find a solution for this? It is happening to me now.

**Comment by TheodoreLizard** [comment]
I got the game on sale on Steam. I don't remember getting a solution for the Epics version.

**Reply by trolles** [reply]
Ok, thanks anyway

**Comment by TheodoreLizard** [comment]
I just got Borderlands 4 on steam. I ran into a similar problem that we both had with Tiny Tina.
Here is what I did to get the settings to save for Borderlands 4:
1) Locate 'Library/Application Support/CrossOver/Bottles/Steam/drive_c/users/crossover' (I configured crossover to install bottles on an external drive, so this would be '/Volumes/<path_to_parent_of_Library>'. I think crossover defaults to home so you might find it in '/Users/<your_home>/Library/...'
2) Locate 'Documents' - this has the little arrow indicating it as a link back to <your_home>/Documents folder. (People familiar with Windows may call this an alias, those on unix will call it a soft-link, etc.)
3) Removed this Documents link.
4) Create a new directory named 'Documents'
5) Restarted the game

I followed the path mentioned in step #1 and opened the directory named 'Documents'. Inside was a directory named 'My games'. 
Command-click on the title 'Documents' which is the titlebar of the finder window showing you the contents of the 'Documents' directory. (I don't know which version of the OS you are using, or on what Mac, so hopefully the terms 'Finder Window', 'Titlebar' and 'Command-click' are understood). A drop-down menu appears and my documents folder is under 'iCloud'. I suspect that saving data to iCloud is having synching issues. Perhaps you write to a file and iCloud syncs it on a schedule but if you try to write to it again too quickly it will fail and give us the message we were seeing. 

After I made the change in steps #3 and #4 I see more directories and files under 'Borderlands 4'. I started a new game and hear Marcus say 'So, you want to hear a story...' or something to that effect. I haven't seen the error message about saving yet so hopefully this works.

I hope this solution works for you and Tiny Tina's Wonderland. I did enjoy playing it and will probably re-visit it again as a different character soon.

**Reply by trolles** [reply]
Hey, thanks for taking the time to write this. I will try the workaround as soon as I have some time. Hopefully I can finally play the game, it looked really fun, although today Hades II is out so I may have to postpone it :).
